### ISSUE TITLE
[Blazor] Not all CSS cursor properties are functional

### ISSUE THREAD

**Issue by Brosinski** [dotnet/aspnetcore#43758]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Assigning certain cursor properties results in no changes being made to the cursor for said element.

The following I tried to use that don't work are:

cursor : col-resize
cursor : move
cursor : cell
cursor : alias

There are many more.

### Expected Behavior

That when a cursor property is assigned to an element, the element will then represent that change by showing the currently assigned property.

### Steps To Reproduce

https://github.com/Brosinski/BrokenCursorBlazor

### .NET Version

6

### Anything else?

Visual Studio 22

**Comment by javiercn** [comment]
@Brosinski thanks for contacting us.

Does this work on a regular webview without Blazor involved?

**Comment by Brosinski** [comment]
Hey @javiercn, thank you for the quick response 

Currently custom cursors don't seem to be available in plain .net MUAI with css styling.

https://github.com/dotnet/maui/issues/4552

Blazor has this functionality for certain cursors already, so I thought it apt to see if the limited range of cursor styles is intended.

Kind Regards

**Comment by javiercn** [comment]
> Blazor has added this functionality for certain cursors already, so I thought it apt to see if the limited range of cursor styles is intended.

I don't think we have added anything in this area, support for this is not specific to Blazor but to whatever the underlying webview/browser supports.
